drafts/2025-03-28-burnout-and-recovery.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+author: joey
+title: Burnout and Recovery
+image: '/img/posts/balance-vs-burnout.jpg'
+date: 2025-04-02
+credit: Nataliya Vaitkevich
+---
+
+Ostensibly, I was doing well. I was managing a small team after being promoted internally, with compensation better than most of my friends in a similar age bracket. I found pinball and rock climbing as hobbies of choice, and I frequently hosted game nights[^1] with friends at my house that I bought not long before housing prices surged.
+
+Burnout is insidious. It’s a slow burn at first that you may not recognize. Eventually a catalyst causes it to explode like gasoline on coals. That catalyst for me (and many others) was the pandemic.
+
+Suddenly, my social hobbies were torn away, my roommate moved back into his parent’s house, I worked from home, and only saw humans in the grocery store and on a screen. I had only work, video games, and home improvements to occupy my time. At work, I was forced by upper management to submit one of my direct reports as a layoff candidate. The raise that I had been promised was put on hiatus, and I was put to work on an impossible project[^2] that got directly in the way of putting time into a machine learning project in which I could see the high long term value[^3].
+
+I had known for a while that I should look for a new job, but was paralyzed by burnout that had spiked from festering to overwhelming. In 2021, when Eric reached out, I was at a low point. My mental and physical health were in the dumps. I had been amazed that such a job existed when Eric joined Apsis years before. Given this incredible opportunity, I felt a lot of internal pressure to impress the team. Thankfully, I was able to push through burnout and earn a spot on the team.
```

Comment:
The last sentence here sounds a bit perilous! It's got a bit of a "burnout is bad but just, y'know, make it work" vibe to it. Any way to soften the blow a bit? Since we haven't actually gotten to the "apsis tries to make this not happen" part yet.

"the promise of etc etc gave me the drive to push through one more time" (but only if it's true!).